gemrb/core/Scriptable/Actor.cpp

```diff
@@ -487,7 +487,7 @@ Actor::Actor()
 	nextWalk = 0;
 	lastattack = 0;
 	InTrap = 0;
-	PathTries = 0;
+	this->resetPathTries();
```

Comment:
Why the explicit "this"? To me it just looks ugly, being redundant.

Reply:
Removed in the following commit. It's just something that I picked up and stuck with for some reason, probably part of my really verbose coding style.